src/rm.js

```diff
@@ -149,6 +149,11 @@ function _rm(options, files) {
     if (stats.isDirectory() && options.recursive) {
       rmdirSyncRecursive(file, options.force);
     }
+
+    // Path is a symbolic link
+    if (stats.isSymbolicLink()) {
```

Comment:
I think the original behavior has been maintained: noop for anything that isn't a directory, file or symlink.